packages/grid/x-data-grid/src/joy/joySlots.tsx

```diff
@@ -48,10 +51,10 @@ function convertVariant<T extends 'outlined' | 'contained' | 'text' | 'standard'
   return (
     variant
       ? {
+          standard: 'outlined',
```

Comment:
@siriwatknp 
Should we unify the variants somehow? 
<img width="532" alt="Screenshot 2023-04-27 at 11 27 49" src="https://user-images.githubusercontent.com/13808724/234820856-45d3150a-dba0-43d8-92e7-db17dde5d2fc.png">

Reply:
Interesting 🤔, Joy UI does not have a standard(underline) variant like Material UI.

I think we can fix it with custom styles in `GridRootStyles`, what do you think?

Reply:
Shouldn't we use the same variant for the selects and textfield visible on the screenshot?
Either outlined or plain?

Reply:
I tried to use `outlined` variant for textfields and selects in [mui/mui-x@`4ada8b7` (#8747)](https://github.com/mui/mui-x/pull/8747/commits/4ada8b7c5c958fcb73db4a11bf6400e5ab02c4ac)
What do you think?

Reply:
Looks good to me!
- would be nice to add some spacing between them but I think it can be a separate PR when we extract the panel
- the date is not aligned. I have created [an issue](https://github.com/mui/material-ui/issues/37129) to fix it from the core side.

<img width="562" alt="image" src="https://user-images.githubusercontent.com/18292247/235539721-e7099ef3-9c53-4dff-81ad-63805603dc92.png">

Reply:
That's weird, it looks aligned for me on deployment preview: https://deploy-preview-8747--material-ui-x.netlify.app/x/react-data-grid/experiments/joy-ui/

<img width="833" src="https://user-images.githubusercontent.com/13808724/235652326-83a9d1ff-da5f-4849-b962-0f025819d384.png">